modules/javafx.controls/src/main/java/com/sun/javafx/scene/control/behavior/TextBinding.java

```diff
@@ -183,25 +183,32 @@ private void parseAndSplit(String s) {
         }
 
         StringBuilder builder = new StringBuilder(s.length());
+        int i = 0;
 
-        for (int i = 0, length = s.length(); i < length; ++i) {
+        // Parse the input string and stop after the first mnemonic.
```

Comment:
This seems to do much more than just fixing numeric mnemonics.  Now it stops after the first mnemonic, whereas it picked the last mnemonic before this change (which seems to be a regression from what happened in jfx16).

I think the regression should be fixed first, then additional features can be added?

Reply:
This PR does not add any additional feature. It restores the behavior of jfx16, where any character is acceptable as a mnemonic, and the first mnemonic is selected.

Reply:
Okay, the wording of the JBS ticket suggested something else, but I see it is just a regression fix.